CharacterCreation/02_Classes.txt

```diff
@@ -427,34 +428,97 @@ Skill Proficiencies:
-	per Quarry. Maximum of Lvl number uses per day. Lasts for the duration of the encounter.
-	At level 10, all damage dealt to this target is increased by 20 (instead of 10).
-	----------------------
-	They Never Saw it Coming -
-	If your Quarry has not seen you for at least 1 turn, your next attack against them does +30 bonus damage.
-	Triggerable once per Quarry.
-	Increases to +50 bonus damage at level 10.
-======================================
+
+Tracker:
+	As an Action, you can study the position and disposition of a target that
+you can see, and indicate it to your Party Members that can communicate with 
+you, either by being on comms with you (see the headset and similar items in
+the "Items/07_Items.txt" document) or that are within 5m of you. Party Members
+(including yourself) that shoot at an indicated target get a +1 to accuracy 
+rolls to hit that target. A target remains indicated until they move, or for
+two Rounds, whichever comes first. You also get 6 skill points in Investigate
+when you take this Class Feat. 
+
+Spotter:
+	Requires that your character be level 10 and that you have the "Tracker"
+class feat. The first time each turn that you track an enemy that you already
+tracked this encounter, you may do so as a Free Action instead of an Action.
+	For example, let's say a Hunter was fighting an enemy robot and an enemy 
+soldier. The first turn, the Hunter spends one Action to track the robot. The
+second turn, the Hunter spends an Action to track the soldier. The third turn 
+the Hunter tracks the robot again, but this time it is a Free Action, so the 
+Hunter still has 4 Actions left on his turn. Continuing his third turn, the 
+Hunter also tracks soldier. Even though the Hunter already tracked the soldier 
+before, the hunter has used up his one Free Tracking Action this turn, and 
+tracking the soldier again costs one Action.
+	In addition, teammates now get a bonus to critically hit on tracked enemies 
+that they hit in accordance to their range and weapon type. This is accomplished 
+by you calling out weak spots that you see on the target and describing shooting 
+conditions such as the bullet drop due to humidity, wind speed and direction, 
+and the coriollis effect making the bullet swerve. The bonus to critically hit 
+is listed below. Note that if the chance to critically hit exceeds the chance to 
+hit, it does not increase the chance to hit; it only garuntees that a hit will 
+be a critical hit.
+
++-------+------------+-----------------------------+
+| Range | Crit bonus | Crit bonus with Long Rifles |
++-------+------------+-----------------------------+
+|   R1  |     5%     |             10%             |
++-------+------------+-----------------------------+
+|   R2  |    10%     |             25%             |
++-------+------------+-----------------------------+
+|   R3  |    20%     |            100%             |
++-------+------------+-----------------------------+
```

Comment:
Do we want these to be written as percentages, or flat bonuses to Critical Range? Right now, this mechanic implies that teammates multiply their Crit range by 1.**, which is a potentially bothersome amount of math-on-the-fly.

Reply:
@bleehu

Reply:
How's that update?